packages/blockfrost/test/blockfrostWalletProvider.test.ts

```diff
@@ -48,6 +48,17 @@ const blockResponse = {
 describe('blockfrostWalletProvider', () => {
   const apiKey = 'someapikey';
 
+  describe('submitTx', () => {
+    it('wraps error in UnknownTxSubmissionError', async () => {
```

Comment:
Fair enough to not be parsing the raw output from `cardano-cli`. It's really up to Blockfrost to provide a more structured response using Ogmios, or at least error code mapping, to result in a better interface for SDK consumers.

```
error {
  error: 'Bad Request',
  message: 'transaction submit error ShelleyTxValidationError ShelleyBasedEraAlonzo (ApplyTxError [UtxowFailure (WrappedShelleyEraFailure (UtxoFailure (OutputTooSmallUTxO [(Addr Testnet (KeyHashObj (KeyHash "0743d16cfe3c4fcc0c11c2403bbc10dbc7ecdd4477e053481a368e7a")) (StakeRefBase (KeyHashObj (KeyHash "06e2ae44dff6770dc0f4ada3cf4cf2605008e27aecdb332ad349fda7"))),Value 1000 (fromList []),SNothing)]))),UtxowFailure (WrappedShelleyEraFailure (UtxoFailure (ValueNotConservedUTxO (Value 1000000000 (fromList [])) (Value 187973 (fromList []))))),UtxowFailure (WrappedShelleyEraFailure (UtxoFailure (OutsideValidityIntervalUTxO (ValidityInterval {invalidBefore = SNothing, invalidHereafter = SJust (SlotNo 37079907)}) (SlotNo 37083558))))])',
  status_code: 400
}
```